core/lib/zksync_core/src/consistency_checker/tests/mod.rs

```diff
@@ -42,8 +42,13 @@ fn create_pre_boojum_l1_batch_with_metadata(number: u32) -> L1BatchWithMetadata
     l1_batch
 }
 
-fn build_commit_tx_input_data(batches: &[L1BatchWithMetadata]) -> Vec<u8> {
-    let commit_tokens = batches.iter().map(L1BatchWithMetadata::l1_commit_data);
+fn build_commit_tx_input_data(
+    batches: &[L1BatchWithMetadata],
+    l1_batch_committer: Arc<dyn L1BatchCommitter>,
```

Comment:
I think this could be:
```suggestion
    l1_batch_committer: &dyn L1BatchCommitter,
```